actiontest/action.yml

```diff
@@ -0,0 +1,28 @@
+name: '1Password Secrets'
```

Comment:
So I could not get act to work properly using the regular action file and the regular test.yml

If you try to run the regular test.yml step that runs on github you'll get a 
```
| Error: Cannot find module '/github/workspace/1password-action/dist/index.js'
```
Lots of people are having the same issue: https://github.com/nektos/act/issues/391, https://github.com/nektos/act/issues/185 
Looks like an issue with the path when it copies. I tried a bunch of things, but one suggestion was to make a copy of your `action.yml` pointing it to `'../dist/index.js'` instead. This seemed to work.

Reply:
I encountered this same problem and got to this PR from a reference in nektos/act#185. This file is not needed (we were doing the same). What needs to happen is that the action needs to add the location path to the checkout. Since the test action is using `./` then checkout path would be: `path: "1password-action"`